polaris.shopify.com/src/components/Markdown/components/DoDont/styles.module.scss

```diff
@@ -2,14 +2,15 @@
 @import '../../../../styles/fonts.scss';
 
 .DoDont {
-  margin: 2rem 0 2rem;
+  #{$notMarginConsideredHarmful} & {
```

Comment:
When the `margin-considered-harmful` class is applied, we opt out of these styles